tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py27,py34,py35,py36,py37,pypy,flake8
+
+[testenv]
+deps = pytest>=3.0
+commands = pytest {posargs:tests}
+
+[testenv:flake8]
+skip_install = true
+deps = flake8
+commands = flake8 pytest_kiwitcms setup.py tests
```

Comment:
I also like to enable pylint, see the TAP plugin for options.

Reply:
Done. What do you think about adding also `isort` and `black`?

Reply:
I like the idea of black but never used it. If you are happy with it (or isort) feel free to include them. I don't have an opinion as long as pylint and flake8 are happy.